lib/data_freshness.rb

```diff
@@ -28,7 +28,93 @@ class DataFreshness
+    convection_assets: { bucket: 'artsy-data', prefix: 'reports/convection.assets/' },
+    convection_offers: { bucket: 'artsy-data', prefix: 'reports/convection.offers/' },
+    convection_partners: { bucket: 'artsy-data', prefix: 'reports/convection.partners/' },
+    convection_partner_submissions: { bucket: 'artsy-data', prefix: 'reports/convection.partner_submissions/' },
+    convection_submissions: { bucket: 'artsy-data', prefix: 'reports/convection.submissions/' },
+    convection_users: { bucket: 'artsy-data', prefix: 'reports/convection.users/' },
+    convection_notes: { bucket: 'artsy-data', prefix: 'reports/convection.notes/' },
+    causality_high_bids: { bucket: 'artsy-data', prefix: 'reports/causality_high_bids/' },
+    causality_calculated_events: { bucket: 'artsy-data', prefix: 'reports/causality.calculated_events/' },
+    causality_events: { bucket: 'artsy-data', prefix: 'reports/causality.events/' },
+    causality_lot_states: { bucket: 'artsy-data', prefix: 'reports/causality.lot_states/' },
+    positron_articles: { bucket: 'artsy-data', prefix: 'reports/positron.articles/' },
+    exchange_orders: { bucket: 'artsy-data', prefix: 'reports/exchange_orders/' },
+    exchange_state_histories: { bucket: 'artsy-data', prefix: 'reports/exchange_state_histories/' },
+    exchange_line_items: { bucket: 'artsy-data', prefix: 'reports/exchange_line_items/' },
+    exchange_transactions: { bucket: 'artsy-data', prefix: 'reports/exchange_transactions/' },
+    exchange_fulfillments: { bucket: 'artsy-data', prefix: 'reports/exchange_fulfillments/' },
+    exchange_line_item_fulfillments: { bucket: 'artsy-data', prefix: 'reports/exchange_line_item_fulfillments/' },
+    exchange_offers: { bucket: 'artsy-data', prefix: 'reports/exchange_offers/' },
+    exchange_fraud_reviews: { bucket: 'artsy-data', prefix: 'reports/exchange_fraud_reviews/' },
+    exchange_admin_notes: { bucket: 'artsy-data', prefix: 'reports/exchange_admin_notes/' },
+    exchange_shipping_quote_requests: { bucket: 'artsy-data', prefix: 'reports/exchange_shipping_quote_requests/' },
+    exchange_quotes: { bucket: 'artsy-data', prefix: 'reports/exchange_shipping_quotes/' },
+    exchange_shipments: { bucket: 'artsy-data', prefix: 'reports/exchange_shipments/' },
+    candela_email_batches: { bucket: 'artsy-data', prefix: 'reports/candela.email_batches/' },
+    candela_emails: { bucket: 'artsy-data', prefix: 'reports/candela.emails/' },
+    marketo_activity_types: { bucket: 'artsy-data', prefix: 'reports/marketo_activity_types/' },
+    marketo_form_submissions: { bucket: 'artsy-data', prefix: 'reports/marketo_form_submissions/' },
+    diffusion_lots_archive: { bucket: 'artsy-data', prefix: 'reports/diffusion.lots_archive/' },
+    diffusion_lots: { bucket: 'artsy-data', prefix: 'reports/diffusion.lots/' },
+    diffusion_source_lots_archive: { bucket: 'artsy-data', prefix: 'reports/diffusion.source_lots_archive/' },
```

Comment:
Neither of the `diffusion.*_archive/` paths seem active. Perhaps they were left over from a migration?

Reply:
yup fixed this.